Cython/Compiler/Symtab.py

```diff
@@ -829,14 +830,17 @@ def declare_cfunction(self, name, type, pos,
         type.entry = entry
         return entry
 
-    def add_cfunction(self, name, type, pos, cname, visibility, modifiers, inherited=False):
+    def add_cfunction(self, name, type, pos, cname, visibility, modifiers,
+                      inherited=False, property=False):
```

Comment:
Maybe call it `is_property`, to avoid the builtin name?

Reply:
ok